src/wallet/test/coinselector_tests.cpp

```diff
@@ -176,8 +176,8 @@ BOOST_AUTO_TEST_CASE(bnb_search_test)
     selection.clear();
 
     // Select 5 Cent
-    add_coin(3 * CENT, 3, actual_selection);
-    add_coin(2 * CENT, 2, actual_selection);
+    add_coin(4 * CENT, 4, actual_selection);
```

Comment:
why these changes?

Reply:
There is an optimal result earlier in the selection so that changes the outcome here.